zookeeper-server/src/test/java/org/apache/zookeeper/test/QuorumBase.java

```diff
@@ -71,13 +79,14 @@
     protected boolean localSessionsEnabled = false;
     protected boolean localSessionsUpgradingEnabled = false;
 
+
     @BeforeEach
     @Override
     public void setUp() throws Exception {
-        setUp(false);
+        setUp(false, true);
     }
 
-    protected void setUp(boolean withObservers) throws Exception {
+    protected void setUp(boolean withObservers, boolean withOracle) throws Exception {
```

Comment:
love the tests! it would be nice if we didn't have to modify every signature with this. (it would have been nice for the withObservers case as well!)